Tests/Contoso.Test.Functional.Droid/Properties/AndroidManifest.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is this an invisible character change? Like end-of-line change or UTF-8 BOM encoding change?

Reply:
https://en.wikipedia.org/wiki/Byte_order_mark it comes and go because of windows vs mac vs where the last time update version script was run.

We could look into this with .gitattributes but not in this pr.